### PR TITLE
Bug/fix animation issues

### LINKED ISSUE #1332: Resize when switch to worker yaml
When changing the tab from `overview` to `yaml` on worker group editor the worker dialog briefly gets smaller. This happens only initially because the yaml editor is not rendered beforehand. Using the `eager` flag for the tab and setting content in advance does not fix the problem.

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
This fixe a couple of layout and animation issues
- The worker configuration dialog no longer gets resized when switching to yaml tab
- Added expand animation to rows and alerts that are added / removed via buttons
- Fixed an issue on shoot editor: The initial modification alert banner did not show up

**Which issue(s) this PR fixes**:
Fixes #1332

**Special notes for your reviewer**:

**Release note**:

```bugfix user
NONE
```

### CODE REVIEW

File: frontend/src/components/ShootWorkers/WorkerConfiguration.vue [hunk 1]
```diff
@@ -126,6 +131,9 @@ export default {
           this.setOverviewData()
         }
         if (value === 'yaml') {
+          // set current height as min-height for yaml tab to avoid
+          // dialog downsize as editor not xet rendered
```
Comment (petersutter) [inline frontend/src/components/ShootWorkers/WorkerConfiguration.vue:135]:
```suggestion
          // dialog downsize as editor not yet rendered
```

File: frontend/src/components/ShootWorkers/WorkerConfiguration.vue [hunk 2]
```diff
@@ -107,7 +111,8 @@ export default {
       networkConfiguration: [],
       networkConfigurationYaml: undefined,
       tabValue: 'overview',
-      editorData: {}
+      editorData: {},
+      overViewTabHeight: 0
```
Comment (holgerkoser) [inline frontend/src/components/ShootWorkers/WorkerConfiguration.vue:115]:
```suggestion
      overviewTabHeight: 0
```

File: frontend/src/components/ShootWorkers/WorkerConfiguration.vue [hunk 3]
```diff
@@ -51,30 +51,34 @@ SPDX-License-Identifier: Apache-2.0
             ref="workerEditor"
             v-on="$workerEditor.hooks"
             hide-toolbar
+            :style="`min-height: ${overViewTabHeight}px`"
```
Comment (holgerkoser) [inline frontend/src/components/ShootWorkers/WorkerConfiguration.vue:54]:
```suggestion
            :style="{ 'min-height': `${overviewTabHeight}px` }"
```

File: frontend/src/components/ShootWorkers/ManageWorkers.vue [hunk 4]
```diff
@@ -300,6 +307,9 @@ export default {
       this.newShootWorkerCIDR = newShootWorkerCIDR
       this.kubernetesVersion = kubernetesVersion
       this.initialZones = uniq(flatMap(workers, 'zones'))
+      this.$nextTick(() => {
```
Comment (holgerkoser) [inline frontend/src/components/ShootWorkers/ManageWorkers.vue:310]: Why do we need this `$nextTick`? Add a comment that explains why it is needed.

File: frontend/src/components/ShootHibernation/ManageHibernationSchedule.vue [hunk 5]
```diff
@@ -253,6 +260,9 @@ export default {
       this.purpose = purpose
       this.parseSchedules(hibernationSchedule)
       this.setNoHibernationSchedule(noHibernationSchedule)
+      this.$nextTick(() => {
```
Comment (holgerkoser) [inline frontend/src/components/ShootHibernation/ManageHibernationSchedule.vue:263]: Why do we need this $nextTick? Add a comment that explains why it is needed.

File: frontend/src/components/ShootWorkers/ManageWorkers.vue (hunk 4 above)
Comment (grolu) [reply frontend/src/components/ShootWorkers/ManageWorkers.vue:310]: added comment

File: frontend/src/components/ShootHibernation/ManageHibernationSchedule.vue (hunk 5 above)
Comment (grolu) [reply frontend/src/components/ShootHibernation/ManageHibernationSchedule.vue:263]: added comment

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**